pkg/lib/cockpit.d.ts

```diff
@@ -23,6 +23,8 @@ declare module 'cockpit' {
     class BasicError {
         problem: string;
         message: string;
+        // only set in src/ssh/cockpitsshrelay.c send_auth_reply(); remove when dropping cockpit-ssh
+        error: string;
```

Comment:
See previous comment.  This isn't right.

Reply:
Dito, see above.